pkg/trust/policy.go

```diff
@@ -54,14 +58,16 @@ type genericRepoMap map[string]json.RawMessage
 
 // DefaultPolicyPath returns a path to the default policy of the system.
 func DefaultPolicyPath(sys *types.SystemContext) string {
+	if sys != nil && sys.SignaturePolicyPath != "" {
+		return sys.SignaturePolicyPath
+	}
+	userPolicyFilePath := filepath.Join(homedir.Get(), userPolicyFile)
```

Comment:
userPolicyFilePath=filepath.FromSlash(filepath.Join(homedir.GetConfigDir(), "containers/policy.json"))

Reply:
Sure thing, let's solve the other conversation first before committing this

Reply:
I think keeping the implementation exactly the same as the c/image one is a minor benefit, but I don’t care that much about this part.

--

But `filepath.FromSlash` must be called _before_ `filepath.Join` for the code to be logically correct. `file path.Join` on a slash-delimited (not platform-delimited) path is incorrect.